x-pack/test/alerting_api_integration/fixtures/plugins/actions/index.ts

```diff
@@ -3,94 +3,28 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-
-import Joi from 'joi';
 import Hapi from 'hapi';
+import { initPlugin as initSlack } from './slack_simulation';
 
 const NAME = 'actions-FTS-external-service-simulators';
 
-export const SLACK_ACTION_SIMULATOR_URI = `/api/_${NAME}/slack`;
+export enum ExternalServiceSimulator {
+  SLACK = 'slack',
+}
 
-interface SlackRequest extends Hapi.Request {
-  payload: {
-    text: string;
-  };
+export function getExternalServiceSimulatorPath(service: ExternalServiceSimulator): string {
+  return `/api/_${NAME}/${service}`;
 }
 
+export const SLACK_ACTION_SIMULATOR_URI = `/api/_${NAME}/slack`;
```

Comment:
shouldn't you just be calling `getExternalServiceSimulatorPath(ExternalServiceSimulator.SLACK)` here - I guess the literal >is< shorter!

Reply:
This export does not appear to be used anywhere unless I'm missing something.  Shouldn't consumers just use `getExternalServiceSimulatorPath`?

Reply:
oops, oversight, once I added that const was supposed to be remove.
I'll remove it.